main.c

```diff
@@ -359,7 +359,7 @@ static void *dofastwork(void *task)
 	ge_scalarmult_base(&ge_public,sk);
 	ge_p3_tobytes(pk,&ge_public);
 
-	for (counter = 0;counter < SIZE_MAX-8;counter += 8) {
+	for (delta = counter = 0;counter < WORK_LOOP_COUNT; counter += 8, delta += 8) {
```

Comment:
this will waste time for non-password cases which don't care about delta at all.
this is very tight loop.
better do it using something like `oldcounter` var which is initially 0 and gets updated upon add and then `addsztoscalar32(sk,counter-oldcounter);`

Reply:
The overhead is not measurable, as most time is spent in ge_add. Likewise for trip count.

Reply:
I bet I could measure it if I ran mkp224o for months.